ReactWindows/ReactNative/Bridge/Queue/MessageQueueThread.cs

```diff
@@ -202,13 +217,13 @@ public SingleBackgroundMessageQueueThread(string name, Action<Exception> handler
                 _queue = new BlockingCollection<Action>();
                 _indicator = new ThreadLocal<bool>();
                 _doneHandle = new ManualResetEvent(false);
-                _asyncAction = ThreadPool.RunAsync(_ =>
```

Comment:
>_asyncAction [](start = 16, length = 12)

I thought there was a reason why we were keeping an awaitable asyncAction here (possibly so we could block on it in the Dispose call).